cypress-io/cypress
# fix: UTM parameters for Latest Runs column

- Closes #22623 

### User facing changelog
* Add UTM parameters to dashboard URL for Latest Runs content

### Additional details
* Update util function to work with URLs that have existing params - previous implementation could end up with duplicate `?` causing a malformed URL if input URL already contained query params like the URLs we get back for direct-linking to the dashboard

### Steps to test
1. Open cloud-connected project with recorded run(s)
2. Login if not already logged in
3. Observe Latest Runs data in Specs Explorer rows, mouse over
4. Inspect - observe that link for Latest Runs "dots" in row and in tooltip that is opened on mouseover have links that contain UTM parameters

### How has the user experience changed?
Latest Runs links to dashboard now contain UTM parameters

### PR Tasks
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
